.github/workflows/aws_test.yml

```
@@ -58,7 +58,7 @@ jobs:
           python-version: 3.11
       
       - name: Install Datalad
-        uses: neuronets/datalad-action/install@main
+        uses: datalad/datalad-action/install@bf-apt-install
```

I am afraid I have mislead us here since I got confused between datalad-installer (where we added that bf-apt-install) and datalad-action.

But now we merged that fix and it got released, so we can just use stock action since it would (I assume) use released datalad-install (now 1.0.4) and it should work or may be at most require `apt-get update` ran before this step.


```suggestion
        uses: datalad/datalad-action/install@master
```